src/DropDownMenu/DropDownMenu.jsx

```diff
@@ -235,10 +189,6 @@ const DropDownMenu = React.createClass({
    */
   getInputNode() {
```

Comment:
@oliviertassinari Sorry! I'm not trying to be difficult :laughing: 

Could you show me which line of code TextField is using this method?